src/python/pants/backend/python/subsystems/pytest.py

```diff
@@ -0,0 +1,36 @@
+# coding=utf-8
+# Copyright 2016 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants.subsystem.subsystem import Subsystem
+
+
+class PyTest(Subsystem):
+  options_scope = 'pytest'
+
+  @classmethod
+  def register_options(cls, register):
+    super(PyTest, cls).register_options(register)
+    register('--pytest-requirements', default='pytest>=2.6,<2.7',
```

Comment:
Why the re-namespacing here?, ie: it seems `s/pytest-//` is in order for all these `--pytest-` prefixed options. The lone remaining long-form `--pytest-unittest2-requirements` vs `--pytest-requirements` seems like it would be clear enough to me coupled with the help strings.